backend/manager/modules/utils/src/main/java/org/ovirt/engine/core/utils/ovf/OvfReaderWriterUtils.java

```diff
@@ -0,0 +1,41 @@
+package org.ovirt.engine.core.utils.ovf;
+
+import org.ovirt.engine.core.common.businessentities.VmDeviceGeneralType;
+import org.ovirt.engine.core.common.utils.VmDeviceType;
+
+public abstract class OvfReaderWriterUtils {
+
+    /**
+     * is special device - device which is managed, but contains the general properties
+     */
+    protected static boolean isSpecialDevice(String device, VmDeviceGeneralType type, boolean includeHostDev) {
```

Comment:
I think, package access level would be better that `protected` here.

Reply:
The same, done.